visual:
  tagSize: 0.10500 #default 0.10
  #camera tag graph interface trigger
  tagGraphOn: true #default false
  #camera trace result trigger
  traceResultOn: false #default true
  #camera trace debug trigger
  traceDebugOn: false #default false
  #detector color mode trigger
  colorOn: false #default false
